src/pack.ts

```diff
@@ -27,6 +27,10 @@ function setFunctionArtifactPath(this: EsbuildPlugin, func, artifactPath) {
 const excludedFilesDefault = ['package-lock.json', 'yarn.lock', 'package.json'];
```

Comment:
@floydspace google functions needs a package.json (and a lockfile i guess) but the node_modules folder is not needed. I think this should take care of google functions, but i cannot test it myself.